pyiceberg/catalog/hive.py

```diff
@@ -135,6 +136,8 @@
 DEFAULT_LOCK_CHECK_MIN_WAIT_TIME = 0.1  # 100 milliseconds
 DEFAULT_LOCK_CHECK_MAX_WAIT_TIME = 60  # 1 min
 DEFAULT_LOCK_CHECK_RETRIES = 4
+DO_NOT_UPDATE_STATS = "DO_NOT_UPDATE_STATS"
+TRUE = "true"
```

Comment:
nit maybe something like 
```suggestion
DO_NOT_UPDATE_STATS_DEFAULT = "true"
```

Reply:
we should always verify if this should be the string `true` or the boolean `True`

Reply:
> we should always verify if this should be the string `true` or the boolean `True`

It should be string `true`, we could verify this in `org.apache.hadoop.hive.common.StatsSetupConst`.

`public static final String TRUE = "true";`

Reply:
> nit maybe something like

thanks, it has been updated